src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -2281,6 +2299,11 @@ private BestSymbolInfo GetBestSymbolInfo(ArrayBuilder<Symbol> symbols, out BestS
 
         private static BestSymbolLocation GetLocation(CSharpCompilation compilation, Symbol symbol)
         {
+            if (symbol is SourceMemberContainerTypeSymbol { IsFile: true })
```

Comment:
>is SourceMemberContainerTypeSymbol { IsFile: true } [](http://example.com/codeflow?start=23&length=51)

Will we get here with a member of a file type? If so, should `BestSymbolLocation.FromFile` be returned in that case?

Reply:
I think we might, in some cases where the member access is not qualified. The only relevant case I can think of, though, involves `using static`.

```cs
// File1.cs
using static C.D;

M();

file class C
{
    public class D
    {
        public static void M() { }
    }
}

// File2.cs
class C
{
    public class D
    {
        public static void M() { }
    }
}
```

I'll add the test. Let me know if any other scenarios come to mind that might hit this issue.

Reply:
I added this test and it "just worked". I suspect this is because we "broke the tie" when binding the `using static C.D`, and binding the call is just a simple lookup within that.